src/org/ensembl/healthcheck/testgroup/ComparaPanIntegrity.java

```diff
@@ -37,6 +37,6 @@ public ComparaPanIntegrity() {
 		addTest(
 			org.ensembl.healthcheck.testcase.eg_compara.ForeignKeyPanMasterTables.class
 		);
-		removeTest(CheckSynteny.class, CheckDuplicatedTaxaNames.class, CheckTopLevelDnaFrag.class, MLSSTagStatsHomology.class, MLSSTagThresholdDs.class, ForeignKeyMasterTables.class, CheckSpeciesSetSizeByMethod.class);
+		removeTest(CheckOrthologQCThresholds.class, CheckSynteny.class, CheckDuplicatedTaxaNames.class, CheckTopLevelDnaFrag.class, MLSSTagStatsHomology.class, MLSSTagThresholdDs.class, ForeignKeyMasterTables.class, CheckSpeciesSetSizeByMethod.class);
```

Comment:
@epaule: You need to import this class before removing it there, e.g: import org.ensembl.healthcheck.testcase.compara.CheckOrthologQCThresholds; at the top of the file.

Reply:
my bad, better?

Reply:
Yes better when we can compile the code @epaule ;-)